vnet/lib/vnet/endpoints/1.0/datapaths.rb

```diff
@@ -41,16 +41,21 @@ def self.put_post_shared_params
   end
 
   param_uuid M::Interface, :interface_uuid, required: true
-  param :broadcast_mac_address, :String, required: true, transform: PARSE_MAC
+
+  # 'broadcast_mac_address' is deprecated and should be phased out.
+  param :mac_address, :String, transform: PARSE_MAC
```

Comment:
With auto-creation of mac addresses we might as well create them for single host cases.

This should probably be revisited when that feature is added.